app/presenters/hyrax/paged_resource_presenter.rb

```diff
@@ -23,7 +23,7 @@ def sequence_rendering
       renderings = []
       if rendering_ids.present?
         rendering_ids.each do |file_set_id|
-          renderings << manifest_helper.build_pdf_rendering(file_set_id)
+          renderings << manifest_helper.build_pdf_rendering(file_set_id) if ESSI.config.dig(:essi, :allow_pdf_download) && current_ability.current_user.admin?
```

Comment:
@labradford Actually, maybe we should add this check to the controller #pdf method too, in case another clever non admins tries to go around the download button.

Reply:
added check to pdf method